app/assets/javascripts/components/school/school_page.es6.jsx

```diff
@@ -49,7 +49,7 @@ class SchoolPage extends Component {
   }
 
   componentDidMount() {
-    SchoolStore.listen(this._listener);
+    this._listener = SchoolStore.listen((state) => this.setState(state));
```

Comment:
We're doing thins a bit differently now, it should be what's on the "red" side of this diff (what it was before)!